src/main/java/org/csanchez/jenkins/plugins/kubernetes/ContainerTemplate.java

```diff
@@ -150,6 +152,13 @@ public void setEnvVars(List<ContainerEnvVar> envVars) {
         this.envVars.addAll(envVars);
     }
 
+    public List<ContainerLivenessProbe> getLivenessProbe() { return livenessProbe; }
```

Comment:
livenessProbe will be null when upgrading from a previous version
You can either check for it and return emptyList or implement readResolve as in https://github.com/jenkinsci/kubernetes-plugin/blob/6b35af2aeebdf81da76a705d1ead94010437c594/src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplate.java#L446